internal/infrastructure/db/models/integration_features.go

```diff
@@ -0,0 +1,13 @@
+package models
+
+type IntegrationFeatures struct {
```

Comment:
Are we using this?

Reply:
Yes, its being used heavily `populate_device_features` https://github.com/DIMO-Network/device-definitions-api/pull/18/commits/7e52deb44f552d8598ac06fce156014064373a0a#diff-e1372ebd61844d8cc4bba372a80c46eeb7de4f26df27b6c040bce1a1f1f643beR65

Reply:
Forgive me, I don’t see the need for a raw query there.